1_slices_modules_and_objects/exercises/binary_search/src/lib.rs

```diff
@@ -1,7 +1,27 @@
 /// Finds the index of `value` in the sorted slice `v`
 /// Returns the index if found, None otherwise
 pub fn binary_search(v: &[i32], value: i32) -> Option<usize> {
-    unimplemented!();
+    let mut high: i32 = v.len() as i32 - 1;
```

Comment:
I think you can keep a `usize` here and simply return early if the length of the vector is 0, but not sure